### DISCUSSION TITLE
ERROR- Downloading Prisma engines for Node-API for debian-openssl-1.1.x

### DISCUSSION THREAD

**Discussion by jhapawan** [prisma/prisma#20353] [Q&A] [answered]
### Bug description

While running the command - npx prisma generate on ubuntu server OS getting the below error

> Downloading Prisma engines for Node-API for debian-openssl-1.1.x [                    ] 0%Error: request to https://binaries.prisma.sh/all_commits/6b0aef69b7cdfc787f822ecd7cdc76d5f1991584/debian-openssl-1.1.x/schema-engine.gz.sha256 failed, reason: read ECONNRESET

### How to reproduce

just simply run the command 
npx prisma init

and getting the error, but this is working on my windows system

### Prisma information

```prisma
// Add your schema.prisma
```

```ts
// Add your code using Prisma Client
```

### Environment & setup

- OS: 
- Database: 
- Node.js version: 

### Prisma Version

```

```

**Comment by janpio** [comment]
Can you access the URL https://binaries.prisma.sh/all_commits/6b0aef69b7cdfc787f822ecd7cdc76d5f1991584/debian-openssl-1.1.x/schema-engine.gz.sha256 manually in a browser on that machine?

**Reply by jhapawan** [reply]
No.

**Reply by janpio** [reply, marked as answer]
Then for some reason your internet connection is blocking the servers we are using. They are hosted on Cloudflare, which is a _very_ common CDN. Is this usually a problem for you?

Can you maybe try a different internet connection, mobile hotspot or VPN, to see if this solves the problem?

**Reply by jhapawan** [reply]
Hi Thank you. 

It is working now for me, previously IPV4 Method was manual, I have set to automatic it start working. 
Might be there was an issue of network.

**Reply by MuhammadAli13562** [reply]
I am having the same problem but the above solution did not solve it

```
npx prisma generate
> Downloading Prisma engines for Node-API for debian-openssl-1.1.x [                    ] 0%Error: request to https://binaries.prisma.sh/all_commits/b9a39a7ee606c28e3455d0fd60e78c3ba82b1a2b/debian-openssl-1.1.x/libquery_engine.so.node.gz.sha256 failed, reason:

```

I can access the site manually and can dowload the binaries , wget works as well but when i run any command with prisma it throws above error

**Comment by victorhsalves** [comment]
> I am having the same problem but the above solution did not solve it
> 
> ```
> npx prisma generate
> > Downloading Prisma engines for Node-API for debian-openssl-1.1.x [                    ] 0%Error: request to https://binaries.prisma.sh/all_commits/b9a39a7ee606c28e3455d0fd60e78c3ba82b1a2b/debian-openssl-1.1.x/libquery_engine.so.node.gz.sha256 failed, reason:
> ```
> 
> I can access the site manually and can dowload the binaries , wget works as well but when i run any command with prisma it throws above error

Did you found a solution to this problem? I'm facing the same thing in a remote ubuntu server.

**Reply by 13217300237** [reply]
me too!

**Comment by MuhammadAli13562** [comment]
Yes.  In my case , my ip was firewalled for some reason. I just connected
to vpn once and disconnected,  errror was gone.
